.github/ISSUE_TEMPLATE/support-ticket.yml

```diff
@@ -0,0 +1,85 @@
+name: Support Ticket
+description: Support ticket for requesting help from Boost team
+title: "[Support Ticket]: "
+labels: ["support", "triage"]
+body:
+- type: checkboxes
+  attributes:
+    label: Boost component
+    description: Please select the boost component you are filing a bug for
+    options:
+      - label: boost daemon - storage providers
+        required: false
+      - label: boost client
+        required: false
+      - label: boost UI
+        required: false
+      - label: boost data-transfer
+        required: false
+      - label: boost index-provider
+        required: false
+      - label: booster-http
+        required: false
+      - label: booster-bitswap
+        required: false
+      - label: LID
```

Comment:
My main concern here is SPs may not know what LID is as it is a new concept. Open to suggestions on what to put here, maybe yugabytedb ? or database?

Reply:
and what is the difference between boostd-data and LID?

Reply:
https://boost.filecoin.io/deployment/boostd-data